editor/src/core/model/element-metadata-utils.ts

```diff
@@ -524,13 +457,13 @@ export const MetadataUtils = {
     let workingFrame: CanvasRectangle = originalFrame
     // If this is held within a group, then we need to add on the frames of the parent groups.
     let ancestorPath = TP.parentPath(target)
-    while (TP.isInstancePath(ancestorPath)) {
-      const ancestorElement = MetadataUtils.getElementByInstancePathMaybe(metadata, ancestorPath)
+    while (!TP.isStoryboardChild(ancestorPath) && !TP.isEmptyPath(ancestorPath)) {
```

Comment:
I'm ever so slightly side-eyeing this, I guess `isInstancePath` was performing something like this check by coincidence?

Reply:
Yeah exactly that (before the natural scenes work)